src/components/events/Event/index.js

```diff
@@ -11,15 +11,16 @@ module.exports = React.createClass({
     name: React.PropTypes.string.isRequired,
     date: React.PropTypes.object.isRequired,
     getStartedUrl: React.PropTypes.string.isRequired,
-    backgroundColor: React.PropTypes.string.isRequired,
     backgroundImageUrl: React.PropTypes.string.isRequired,
     supporterCount: React.PropTypes.number.isRequired,
+    width: React.PropTypes.string.isRequired,
     i18n: React.PropTypes.object,
   },
 
   getDefaultProps: function() {
     return {
       defaultI18n: {
+        backgroundColor: '#525252',
```

Comment:
This doesn't belong in the internationalisation. Just leave it as a separate prop.